pandas/tests/plotting/frame/test_frame_subplots.py

```diff
@@ -422,11 +419,6 @@ def test_subplots_dup_columns_secondary_y_no_subplot(self):
         assert len(ax.lines) == 0
         assert len(ax.right_ax.lines) == 5
 
-    @pytest.mark.xfail(
-        np_version_gte1p24 and is_platform_linux(),
-        reason="Weird rounding problems",
-        strict=False,
-    )
```

Comment:
Ran these a bunch locally and couldn't get them to fail. Also tried putting in values very near 0.0 and still could not.